### PR TITLE
update r23 items

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # .

#### Changes proposed in this Pull Request:
- updating R23 roadmap items
- add r24 roadmap items
- 

#### To Test:
-

### CODE REVIEW

**Review by emclaug2 — COMMENTED** [review]
You'll need to run `yarn start` to build the updated JSON file for R23.

**Review by joebochill — COMMENTED** [review]
Everything in R23 should be either marked as Finished or Deferred. I think in this situation everything can be considered finished.

File: src/data/doc-it/R25Roadmap.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    type: ItemTypeFilter;
+    framework?: FrameworkFilter[];
+    items: RoadmapItem[];
+};
+
+export const roadmap: RoadmapBucket[] = [
+    {
+        name: 'Adopter communication',
+        description: 'Improve adopter communications',
+        type: 'development',
```
Comment (huayunh) [inline src/data/doc-it/R25Roadmap.ts:31]: type all

File: src/data/doc-it/R24Roadmap.ts [hunk 2]
```diff
@@ -26,14 +26,14 @@ type RoadmapBucket = {
...
+        name: 'Update Component Documentation',
+        description: 'Design new component documentation for Brightlayer-UI',
+        type: 'design',
+        framework: ['all'],
         items: [
             {
-                name: 'Visualization Components',
-                description: 'Reusable components for common visualization elements',
+                name: 'Component doc design',
+                description: 'Create component documentation in Figma',
```
[the remark this answers is not in the document]
Comment (jeffvg) [reply src/data/doc-it/R24Roadmap.ts:36]: yes I believe it is. Are you seeing a Drop ionic somewhere?

File: src/data/doc-it/R24Roadmap.ts [hunk 3]
```diff
@@ -42,8 +42,8 @@ export const roadmap: RoadmapBucket[] = [
         ],
     },
     {
-        name: 'Figma Design Patterns',
-        description: 'Adding design pattern examples to Figma',
+        name: 'Data Visualization Component Designs',
+        description: 'Adding data visualization designs to Figma',
```
[the remark this answers is not in the document]
Comment (jeffvg) [reply src/data/doc-it/R24Roadmap.ts:46]: good catch

File: src/data/doc-it/R24Roadmap.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (huayunh) [reply src/data/doc-it/R24Roadmap.ts:36]: That was on the top of the file, I believe. Just a small thing

File: src/data/doc-it/R24Roadmap.ts [hunk 4]
```diff
@@ -26,14 +26,14 @@ type RoadmapBucket = {
 
 export const roadmap: RoadmapBucket[] = [
     {
-        name: 'Angular Component Library',
-        description: 'Reusable UI elements from @brightlayer-ui/angular-components',
-        type: 'development',
-        framework: ['angular'],
+        name: 'Update Component Documentation',
```
[the remark this answers is not in the document]
Comment (jeffvg) [reply src/data/doc-it/R24Roadmap.ts:29]: Updated the design names and descriptions.
Changed out the bootcamp.
Moved iconography to r25.
Added Redesign dev component documentation

**Comment by jeffvg** [comment]
@huayunh can you give this another review again

**Review by huayunh — APPROVED** [review]

**Pull request merged**